packages/clerk-js/src/core/resources/Organization.test.ts

```diff
@@ -8,8 +8,8 @@ describe('Organization', () => {
       name: 'test_name',
       public_metadata: { public: 'metadata' },
       slug: 'test_slug',
-      logo_url: 'https://url-for-logo.png',
-      image_url: 'https://clerk.com',
+      image_url:
```

Comment:
Nit: Formatting

Reply:
This change is made by the `npm run format` script based on our rules.

Reply:
Seems weird 🤔